compiler/modules/types/core.bal

```diff
@@ -424,6 +429,7 @@ public final BasicTypeBitSet FUTURE = basicType(BT_FUTURE);
 
 // this is SubtypeData|error
 public final BasicTypeBitSet TOP = basicTypeUnion(BT_MASK);
+public final BasicTypeBitSet TOP_OR_UNDEF = TOP | UNDEF;
```

Comment:
This terminology is a bit confusing.  TOP usually means everything. Let's discuss.

Reply:
All these _OR_UNDEF names are a bit unwieldy.